dora/core/server/master/src/main/java/alluxio/master/job/LoadJobFactory.java

```diff
@@ -60,11 +62,23 @@ public Job<?> create() {
         .ofNullable(AuthenticatedClientUser.getOrNull())
         .map(User::getName);
 
+    Predicate<UfsStatus> predicate = Predicates.alwaysTrue();
```

Comment:
These logic should also appear at JournaledLoadJobFactory so we can recover from journal entry

Reply:
Updated. I put these logic into JournaledLoadJobFactory.